packages/x-date-pickers/src/CalendarPicker/DayPicker.tsx

```diff
@@ -74,17 +77,41 @@ export interface DayPickerProps<TDate>
   gridLabelId?: string;
 }
 
+const useUtilityClasses = (
+  ownerState: DayPickerProps<any> & { classes?: Partial<DayPickerClasses> },
+) => {
+  const { classes } = ownerState;
+  const slots = {
+    header: ['header'],
+    weekDayLabel: ['weekDayLabel'],
+    loadingContainer: ['loadingContainer'],
+    slideTransition: ['slideTransition'],
+    weeksContainer: ['weeksContainer'],
```

Comment:
Not sure it's semantically the best. But, it's confusing the `week`/`weeks`

```suggestion
    monthContainer: ['monthContainer'],
```

Reply:
Very good suggestion 🙈 👍